website/src/components/SingleFeature/index.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import { ReactNode, useState } from "react";
+import { HiCloud, HiMiniPuzzlePiece, HiLockClosed } from "react-icons/hi2";
+import { SlideIn } from "@/components/Animations";
+import { manrope } from "@/lib/fonts";
+import ActionLink from "../ActionLink";
+
+interface SingleFeatureProps {
+  title: string;
+  boldedTitle?: string;
+  desc: string;
+  link: string;

```

Comment:
@Patticatti The reason this is failing typecheck is because you need to use `Route<string>` type for hrefs because we have staticRoutes turned on in NextJS.

This validates all hrefs so that it's not possible to create broken links within the site.
```suggestion
  link: Route<string>;

```